platform/engineering/code_review_guidelines.md

```diff
@@ -50,6 +50,21 @@ When your code is reviewed, here are some of the things we're looking for:
     * If you programmatically ignore a linting recommendation (e.g. via `rubocop:disable`), please comment in the PR explaining why.
     * If your linter does raise any issues unrelated to the PR's subject matter, please fix these in a separate PR.
 
+
```

Comment:
I don't think this file is where we should put this policy. Partly a "keeping this one as to-the-point and broadly accessible as possible", partly because it's more for us (VSP) than it is for our users (VFS), which this doc is written towards.

I don't have a great location proposal though, closest I've got would be a file somewhere within https://github.com/department-of-veterans-affairs/va.gov-team/tree/master/teams/vsp/teams/tools/backend ?